src/pages/layout/Search/Index.tsx

```diff
@@ -0,0 +1,132 @@
+import React, {useState, useEffect, useMemo} from "react";
+import {throttle} from "lodash";
+import {Autocomplete, AutocompleteInputChangeReason} from "@mui/material";
+import SearchTransaction from "./SearchTransaction";
+import SearchAccount from "./SearchAccount";
+import SearchInput from "./SearchInput";
+import ResultPaper from "./ResultPaper";
+import SearchResultNotFound from "./SearchResultNotFound";
+
+const HEX_REGEXP = /^(0x)?[0-9a-fA-F]+$/;
```

Comment:
there is this function, https://github.com/aptos-labs/explorer/blob/main/src/pages/utils.tsx#L143 , it is a different checking but the naming can confused (one of them need to be changed...?)

Reply:
`isHex` is not exact accurate because it returns `false` for inputs like `1` which is a valid address and version. I have a task in my TODO list to unify this https://linear.app/aptoslabs/issue/WP-169/[explorer]-0x-address-format. Will revisit it later with this task.